tox.ini

```diff
@@ -41,16 +41,15 @@ commands =
     python -m OpenSSL.debug
     python -m twisted.trial -j4 --reporter=text twisted
 
-[testenv:flake8]
+[testenv:lint]
 basepython = python3
 deps =
     black
-    flake8
-    flake8-import-order
+    ruff==0.0.284
```

Comment:
This is literally our only pin, and we have no mechanism for bumping. We need to either unpin or do this in a dependabot friendly way IMO.

Reply:
We have a pin for mypy in the exact same way here. It definitely isn't dependabot friendly though, I agree 😄

Reply:
Its also not local dev friendly (since I know neither of us tox)